src/components/ViewerTitle.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { Info } from './Icons';
+import { ModelInfo } from '../constants';
+
+import '../../css/viewer_title.css';
+interface ViewerTitleProps {
+  modelInfo: ModelInfo;
+}
+
+const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
+  props: ViewerTitleProps
+): JSX.Element => {
+  const hasMetaData = (): boolean => {
+    for (const key in props.modelInfo) {
+      if (
+        key !== 'title' &&
+        props.modelInfo[key as keyof ModelInfo] !== undefined
+      ) {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  return (
+    <div className="title-container">
+      <div className="title">
+        {props.modelInfo.title}
+        {hasMetaData() ? <div className="info-button"> {Info} </div> : null}
```

Comment:
Nit: I know from the description that you're going to implement this later, but it might be nice to add a TODO for that in the code itself

Reply:
Done, thanks